recipes/bio_hansel/meta.yaml

```diff
@@ -24,15 +25,15 @@ requirements:
     - pandas >=0.20.1
     - attrs ==17.2.0
     - pyahocorasick >=1.1.6
+    - libgcc >=5.2.0 # [linux]
```

Comment:
libgcc does not need to be a build dependency

Reply:
Okay, removed in a25f6c4. Thanks!